### DISCUSSION TITLE
Include override without unmatching the existing path

### DISCUSSION THREAD

**Discussion by nataliescottdavidson** [BurntSushi/ripgrep#1652] [General]
When I use a glob override, WalkBuilder no longer builds the existing path structure. It treats the overrrides as whitelist-only structure. I ran into this problem by [trying to include `.well-known` dotfiles but no other hidden files.](https://github.com/cloudflare/wrangler/issues/980) Is matching like this possible?

**Comment by BurntSushi** [comment]
I don't understand your question, sorry. Could you please provide an example program, inputs, actual outputs and desired outputs?

**Comment by nataliescottdavidson** [comment]
Sorry this isn't a minimum reproducible example, but hopefully [this diff](https://github.com/nataliescottdavidson/wrangler/pull/1/files#diff-ad205eef9d2d7321e31da6507b0972fd) illustrates whatis going on a little better. The expected output is the previously matched set of paths + .well-known. What I get is just the .well-known path.

**Reply by BurntSushi** [reply]
Sorry, but that doesn't really help. What I'd really like to see is:

1. A small Rust program that I can compile and execute.
2. A set of inputs (perhaps some shell commands to create a directory tree).
3. The actual output.
4. The desired output.

**Reply by nataliescottdavidson** [reply]
Ok, I've reproduced a small example here. https://github.com/nataliescottdavidson/walkbuilder-example

**Comment by EverlastingBugstopper** [comment]
Hey @BurntSushi - is this example enough to reproduce?
